### ISSUE TITLE
FXDK Blue Game Screen and Game Wont Start

### ISSUE THREAD

**Issue by RJMGames** [citizenfx/fivem#2580]
### What happened?

I've used fxdk for a while and have had no issues... I came back to it the other day and now the game won't start in fxdk. I've tried uninstalling and clearing all five m data and re installing... and I have made sure I have all the requirements installed I just can't figure out what is wrong I can't load the map editor either. I have also tried re installing GTA using the epic games version and steam version.
<img width="959" alt="image" src="https://github.com/citizenfx/fivem/assets/95596423/19cec609-d1e6-4f38-9c99-00863afaba2e">

### Expected result

The game should start in the game window

### Reproduction steps

1. I start fxdk
2. Create a fresh project
3. Press the play button
4. nothing happens and clicking the game view manually shows blue screen

### Importancy

Crash

### Area(s)

FiveM, FxDK

### Specific version(s)

FXDK Version: 1.63.2-cfx

### Additional information

I usually use c# in fxdk. I've used the software in the past with no issues, so I know how to use it.

**Comment by Franklinspeeder** [comment]
Same here. Also World Editor loading time is slow.

**Comment by Franklinspeeder** [comment]
https://imgur.com/a/dlXRr1K

**Comment by RJMGames** [comment]
The issue isn't that... the issue is the game won't start

**Comment by Franklinspeeder** [comment]
I just happen to have 2 issues

**Comment by RJMGames** [comment]
sorry read ur comment wrong from my phone... Ya i have the same two issues game not loading and map editor not loading at all for me

**Comment by gottfriedleibniz** [comment]
Thanks for the report. We recently dropped non-ASLR builds and forgot to [update](https://github.com/citizenfx/fivem/blob/edf4a88c18d93c53c7b9bf5199f58e90367d9703/code/components/fxdk-main/src/SDKGameProcessManager.cpp#L53) the SDK accordingly,

**Comment by Ciccrts** [comment]
This market Capitals trade are real scammers. I made a promise to share my scam encounter. And how I got back my money from this scam trade broker . I could never have imagined falling for the persuasion of such scammers, but it happened. I transferred almost Є72,620 to their fake trading account in my name, with no access to withdraw my money out, to my greatest surprise, I got in contact with an expert hacker, robertslee 6 1 8. @. gmail.  .com  Or on WhatsApp (+1 856 549 7469) to help me retrieve my money back from the scammer which was successfully refunded back to me, I’m thankful to the recovery expert and the organization for helping me recover my money back successfully. I guess someone might find this information useful.
